src/callbacks_step/amr_dg1d.jl

```diff
@@ -23,6 +23,20 @@ function refine!(u_ode::AbstractVector, adaptor, mesh::TreeMesh{1},
     GC.@preserve old_u_ode begin # OBS! If we don't GC.@preserve old_u_ode, it might be GC'ed
         old_u = wrap_array(old_u_ode, mesh, equations, dg, cache)
 
+        # Compute mean values for the elements to be refined
+        # Only if limiter was passed
+        if limiter! !== nothing
+            u_mean_refined_elements = Matrix{eltype(u_ode)}(undef,
+                                                            nvariables(equations),
+                                                            length(elements_to_refine))
```

Comment:
I am not sure if it makes a difference, but maybe just using a plain `Array` is sufficient here, as no advanced operations are performed on this datastructure.